### ISSUE TITLE
iterations_per_loop explanation

### ISSUE THREAD

**Issue by working12** [google-research/text-to-text-transfer-transformer#414]
iterations_per_loop:  from the source itself is `integer, steps per train loop`. Now steps means each gradient update. So that means if I have batch size of 128. Then it will read 128 Q/A pairs and then will update once.

But where does this training loop come form? Can anybody explain in simple terms?

**Comment by craffel** [comment]
Hi, iterations_per_loop defines the number of training steps that should be run at a time on the TPU. It is an implementation detail that can largely be ignored.
